guides/common/Makefile

```diff
@@ -68,7 +68,7 @@ $(IMAGES_TS): $(IMAGES)
 	@touch $(IMAGES_TS)
 
 $(DEST_HTML): $(SOURCES) $(DEPENDENCIES) $(DOCINFO_SOURCES)
-	bundle exec asciidoctor -a build=$(BUILD) $(BASEURL_ATTR) -a docinfo=shared -a docinfodir=common -a date_mdy="$(DATE_MDY)" -a date_my="$(DATE_MY)" -a docdatetime="$(DATE_MY)" -a stylesheet=$(BUILD).css -a linkcss=true -b html5 -d book --trace -o $@ $< 2>&1 | tee $@.log
+	bundle exec asciidoctor -a build=$(BUILD) $(BASEURL_ATTR) -a docinfo=shared -a docinfodir=common -a date_mdy="$(DATE_MDY)" -a date_my="$(DATE_MY)" -a docdatetime="$(DATE_MY)" -a stylesheet=$(DEST_DIR)/$(BUILD).css -b html5 -d book --trace -o $@ $< 2>&1 | tee $@.log
```

Comment:
This makes sense but I think you should ensure it's present prior to building. #1013 does that.

Reply:
Yeah I had this change locally but then it built fine... weird. Anyway, incorporated into my patch now to prevent conflicts.

Reply:
I am going even simplify it more, the phony `css` is no longer needed actually. It was a leftover.

Reply:
Oh yes, that's even better. Always great when you can achieve more with fewer lines of code :)